src/tamp_improv/approaches/rl_blocks2d_improvisational_tamp_approach_old.py

```diff
@@ -0,0 +1,112 @@
+"""RL-based implementation of improvisational TAMP for Blocks2D environment."""
```

Comment:
if this is an old file, can we remove it? will it remain in the git history if we ever need it